go/vt/sqlparser/ast.go

```diff
@@ -734,6 +734,9 @@ type DDL struct {
 	OptLike       *OptLike
 	PartitionSpec *PartitionSpec
 
+	// IsVitessSequence is set for AddVschemaTableStr and refers to whether the table is a sequence table.
+	IsVitessSequence bool
```

Comment:
I would just call it “IsSequence”. We know it’s Vitess. :-)

Reply:
I think you can have SQL sequences as well. For instance MariaDB allows "CREATE SEQUENCE" (https://mariadb.com/kb/en/library/create-sequence/). Initially wanted to differentiate.

Reply:
I see. I think we don't need to support MariaDB sequences. Wdyt @sougou ?

Reply:
IMO we don't need a special bit for this -- you can just re-use the VindexSpec and make the vindex "type" equal to "sequence.